CONTRIBUTING.md

```diff
@@ -100,6 +101,50 @@ Be sure to include any related GitHub issue references in the commit message.  S
 [GFM syntax](https://guides.github.com/features/mastering-markdown/#GitHub-flavored-markdown) for referencing issues
 and commits.
 
+
+[dronevic]:https://ci.vmware.run/vmware/vic
+[e2edronevic]:https://e2e.ci.vmware.run/vmware/vic
+[dronesrc]:https://github.com/drone/drone
+[dronecli]:http://readme.drone.io/devs/cli/
+
+## Automated Testing
+
+Automated testing uses [Drone][dronesrc].
```

Comment:
Does ROBOT they require drone?  Can i run this through docker locally without drone?  I ask because we have a build step which builds all of our binaries via docker (which is very convenient).  Can we add the ROBOT stuff to the `makefile` so it's a buildstep which *doesn't* require drone?